panel/repl.html

```diff
@@ -40,10 +40,26 @@ <h3 class="type--standard-weight">Settings</h3>
         <div class="settings__spinner"></div>
         <div class="settings__content">
           <div class="settings-column">
-            <h4 .>Transformer</h4>
+            <h4>Transformer</h4>
             <div class="transformer-options"></div>
           </div>
           <div class="settings-column">
+            <h4>Editor</h4>
+            <div class="settings__option-container">
+                <div>tabSize</div>
```

Comment:
These should probably be human-readable titles, i.e. `Tab size` and `Indent with tabs`